src/qt/walletframe.cpp

```diff
@@ -145,6 +145,13 @@ void WalletFrame::gotoSendCoinsPage(QString addr)
         i.value()->gotoSendCoinsPage(addr);
 }
 
+void WalletFrame::gotoPrivateSendCoinsPage(QString addr)
+{
+    QMap<QString, WalletView*>::const_iterator i;
+    for (i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i)
+        i.value()->gotoPrivateSendCoinsPage(addr);
```

Comment:
add brackets, use auto for iterator
```suggestion
    for (auto i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i) {
        i.value()->gotoPrivateSendCoinsPage(addr);
    }
```